CHANGELOG.md

```diff
@@ -2,4 +2,5 @@
 
 * [CHANGE] Removed global metrics for KV package. Making a KV object will now require a prometheus registerer that will
   be used to register all relevant KV class metrics. #22
-* [CHANGE] Added CHANGELOG.md and Pull Request template to reference the changelog
\ No newline at end of file
+* [CHANGE] Added CHANGELOG.md and Pull Request template to reference the changelog
+* [ENHANCEMENT] Add 'pkg/util/http' http utilities from cortex #34
```

Comment:
Wondering if it might be better to refer to the package name it gets in dskit, as opposed to the one in Cortex (pkg/util/http) to avoid confusion on part of the reader?